### PR TITLE
[MD-82] DB Setup

### PR DESCRIPTION
**Pull request by IamKittitat**
**PR (Pull Request) Naming**
Please name the PR title as _"[MD-Number] Title"_

## What did you do in this PR ?
- DB Schema 
- Mock Data
- Connect node with pg

### CODE REVIEW

File: backend/src/index.ts [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
 const port = process.env.PORT ?? 8000
+const db = new Client({
+  user: process.env.PG_USER,
+  host: process.env.PG_HOST,
+  database: process.env.PG_DATABASE,
+  port: parseInt(process.env.PG_PORT ?? "5432"),
+  password: process.env.PG_PASSWORD
+})
+
+db.connect()
```
Comment (IamMrTeapot) [inline backend/src/index.ts]: Please don't connect to db in index.ts kub,better move to repository

File: backend/src/index.ts [hunk 2]
```diff
@@ -21,4 +31,4 @@ app.listen(port, () => {
 
 app.use("/example", exampleRouter)
 
-export default app
+export { app, db }
```
Comment (IamMrTeapot) [inline backend/src/index.ts]: index.tsx should export only one module

**Review by IamMrTeapot — CHANGES_REQUESTED** [review]

File: backend/src/index.ts (hunk 1 above)
Comment (IamMrTeapot) [reply backend/src/index.ts]: JK, I think move to config/pgdbConnnection.ts is better

Comment (IamKittitat) [reply backend/src/index.ts]: Why shouldn't connect in index.ts? and if moved to repo isn't it will duplicate code of connecting the db?

Comment (IamMrTeapot) [reply backend/src/index.ts]: //You will need to call it in index.ts anyways but please move the function to config

Comment (IamMrTeapot) [reply backend/src/index.ts]: Not sure but I believe that we will need to use db variable in the repository too, maybe for querying? Anyways it makes the code a little bit less coupling

**Review by IamMrTeapot — APPROVED** [review]

**Pull request merged**